.github/workflows/provision-destroy.yaml

```
@@ -1,19 +1,23 @@
-name: Deploy Infrastructure
+name: Infrastructure Management
 
 on:
   push:
     branches:
-      - setup-infra2
+      - setup-infra
   workflow_dispatch:
     inputs:
-      infra-deploy:
+      provision:
         required: true
```

If `required=true`, there is no need to provide a `default`. I suppose this could also be `required=false`, because when an admin wants to `destroy`, he/she can surely omit this option.
```suggestion
        required: false
```

---

But I want to leave option `required: true` to make sure admin know what they are doing